flujo/validation/fixers.py

```
@@ -0,0 +1,425 @@
+    new_text = yaml_text.replace(target, "previous_step | tojson")
+    return new_text, count
+
+
+def count_findings(report: object, rule_id: str) -> int:
+    try:
+        errs = getattr(report, "errors", []) or []
+        wrns = getattr(report, "warnings", []) or []
+        return sum(1 for e in errs if getattr(e, "rule_id", None) == rule_id) + sum(
+            1 for w in wrns if getattr(w, "rule_id", None) == rule_id
+        )
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)